py_expression_eval/__init__.py

```diff
@@ -227,6 +227,34 @@ class Expression(Expression):
     CALL         = 128
     NULLARY_CALL = 256
 
+    ops = [('+', 2, '+'),
+           ('-', 2, '-'),
+           ('**', 6, '**'),
+           ('*', 3, '*'),
+           (u'\u2219', 3, '*'), # bullet operator
+           (u'\u2022', 3, '*'), # black small circle
+           ('/', 4, '/'),
+           ('%', 4, '%'),
+           ('^', 6, '^'),
+           ('||', 1, '||'),
+           ('==', 1, '=='),
+           ('!=', 1, '!='),
+           ('<=', 1, '<='),
+           ('>=', 1, '>='),
+           ('<', 1, '<'),
+           ('>', 1, '>'),
+           ('and ', 0, 'and'),
+           ('or ', 0, 'or')]
+
+
+    def register_operation(self, token, priority, index, func):
+
+        if token not in self.ops1 or token not in self.ops1:
```

Comment:
um.... this seems wrong... same check twice? And not checking ops ?

Reply:
Yeah, you guys are right. It worked for my simple cases, but indeed there is a lack of errorhandling and unittests. I'am sorry, about that. 
Unfortunately I'am currently quite swamped at work and won´t be able to fix this right away. You guys have to give me some time, please.